src/widget/contentdialog.cpp

```diff
@@ -360,6 +362,28 @@ void ContentDialog::cycleContacts(bool forward, bool loop)
     }
 }
 
+void ContentDialog::onVideoShowed(QSize size)
```

Comment:
It's "onVideoShown". Why not simply "onVideoShow / onVideoHide" -> Less confusion.